meetupselector/api/routes/talks.py

```diff
@@ -1,6 +1,22 @@
+import logging
+import typing as t
+
 from ninja import Router
+from pydantic import UUID4
+
+from meetupselector.api.schemas.talks import TopicListSchema, TopicRetrieveSchema
+from meetupselector.talks.services import TopicService
+
+logger = logging.getLogger(__name__)
```

Comment:
Non blocking: I was wondering to know why do we initialize this logger here. Seems to be unused, isn't it?